plugins/omclickhouse/omclickhouse.c

```diff
@@ -431,8 +431,9 @@ computeBulkMessage(const wrkrInstanceData_t *const pWrkrData,
 	const uchar *const message, char **newMessage)
 {
 	size_t r = 0;
-	if(pWrkrData->batch.nmemb != 0) {
-		*newMessage = strchr(strstr((const char*)message, "VALUES"), '(');
+	char *v;
+	if(pWrkrData->batch.nmemb != 0 && (v = strstr((const char *)message, "VALUES")) != NULL) {
+		*newMessage = strchr(v, '(');
```

Comment:
thx for the PR. But while we are at it, I think this is an additional potential NULL pointer reference in the case `'('` is not present. Would you like to add the check here as well?

Reply:
ok, soon

Reply:
@rgerhards  done.